leaf-ui/js/displayAnalysis.js

```diff
@@ -320,12 +313,36 @@ function addAnalysisConfig() {
     $(".result-elements").css("background-color", "");
     var buttonLabel = currAnalysisConfig.id + "-button";
     var label = currAnalysisConfig.id + "-dropdown";
-    $("#analysis-sidebar").append(
-        '<button class="log-elements" id="'+currAnalysisConfig.id+'" style="padding: 12px; font-size: 16px; border: none; outline: none; background-color:#A9A9A9">' + currAnalysisConfig.id + '</button><div style="position:absolute; display:inline-block"><button class="dropdown" id= "'+buttonLabel+'" style="padding: 12px; font-size: 16px; height: 42px; border: none; outline: none;"><i class="fa fa-caret-down fa-2x" style="cursor:pointer;"></i></button>'
+    // Add a larger div to contain the configuration "<config id>-container"
+    $("#configurations").append(
+        '<div class = "analysis-configuration" id="' + currAnalysisConfig.id + '-container">'
+        + '<button class="log-elements" id="'+currAnalysisConfig.id+'" style="padding: 12px; font-size: 16px; border: none; outline: none; background-color:#A9A9A9">' + currAnalysisConfig.id + '</button><div style="position:absolute; display:inline-block"><button class="dropdown" id= "'+buttonLabel+'" style="padding: 12px; font-size: 16px; height: 42px; border: none; outline: none;"><i class="fa fa-caret-down fa-2x" style="cursor:pointer;"></i></button>'
         + '</div><div class = "dropdown-container" id="'+label+'"></div>'
+        + '</div>'
       );
 }
 
+/**
+ * Removes an analysis configuration from the analysisMap and UI sidebar
+ * TODO implement with a button
+ */
+function removeConfiguration() {
+    // Remove full configuration div (includes results)
+    var configDiv = document.getElementById(currAnalysisConfig.id);
+    configDiv.remove();
+    // Remove config from analysisMap
+    analysisMap.delete(currAnalysisConfig.id);
+}
+
+/**
+ * Clears the analysis config sidebar
+ */
+function clearAnalysis() {
```

Comment:
This function name might be a little vague, with all the other types of analysis. Maybe `clearAnalysisConfigSidebar()`? Wordy, I know

Reply:
changed! :)